packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -469,6 +469,32 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  factory CookieManager() {
+    if (_instance == null) {
+      final MethodChannel methodChannel =
+          const MethodChannel('plugins.flutter.io/cookie_manager');
+      _instance = CookieManager.private(methodChannel);
+    }
+    return _instance;
+  }
+
+  CookieManager.private(this._channel);
+
+  static CookieManager _instance;
+  final MethodChannel _channel;
+
+  /// Clears all cookies.
+  ///
+  /// This is supported for >= IOS 9 and Android api level >= 16.
+  /// returns true if cookies were present before clearing, else false.
+  Future<bool> clearCookies() => _channel
+      // ignore: strong_mode_implicit_dynamic_method
```

Comment:
Add the same comment we have everywhere(minus the typo 😄 ) so it's easy to find all of these at once (and we need a TODO here anyway)

Reply:
keep it TODO(amirh), most likely someone will end up searching for all these `TODO(amirh):` occurrences at once.